package.json

```diff
@@ -79,26 +79,26 @@
     "@vue/eslint-config-typescript": "^14.5.0",
     "@vue/language-plugin-pug": "^2.2.10",
     "eslint-config-prettier": "^10.1.5",
-    "eslint-import-resolver-typescript": "^4.3.4",
+    "eslint-import-resolver-typescript": "^4.3.5",
     "eslint-plugin-import": "^2.31.0",
     "fishery": "^2.3.1",
     "globals": "^16.1.0",
     "jsdom": "^26.1.0",
     "postcss-html": "^1.8.0",
     "prettier": "^3.5.3",
-    "quicktype": "^23.1.4",
-    "sass": "^1.88.0",
+    "quicktype": "23.2.4",
```

Comment:
Locking to this version to avoid this bug: https://github.com/glideapps/quicktype/issues/ 2753 .